addons/crm/tests/test_crm_lead_convert_mass.py

```diff
@@ -166,7 +166,7 @@ def test_mass_convert_performances(self):
         test_leads = self._create_leads_batch(count=50, user_ids=[False])
         user_ids = self.assign_users.ids
 
-        with self.assertQueryCount(user_sales_manager=1038):  # 1038 generally on runbot - crm only: 1029
+        with self.assertQueryCount(user_sales_manager=1042):  # 1042 generally on runbot - crm only: 1029
```

Comment:
Hello,

Why this change ? How is crm only affected (also update comment if necessary) ?

Reply:
Hello!
I ran some tests locally using runbot's DB, as well as a clean DB with CRM only, and updated the values accordingly.
I guess there have been some optimizations elsewhere, because the new values are lower than before :wink: 
Should they be adjusted then or just left as is ?

Reply:
Beware this test is quite random... if it is lower I wouldn't change it. Some PRs are coming with performance update (notably from juc), I would let it as it is currently. If it is higher, well ... find extra queries and try to avoid them :p .